packages/slice-machine/components/DeleteDocumentsDrawer/HardDeleteDocumentsDrawer.tsx

```diff
@@ -57,14 +45,14 @@ export const HardDeleteDocumentsDrawer: React.FunctionComponent<{
 
   return (
     <SliceMachineDrawerUI
-      isOpen={isDeleteDocumentsDrawerOpen}
+      isOpen={modalData.type === "HARD"}
```

Comment:
Should we be using an enum rather than the string value?

Reply:
If TS can infer it, a string value is perfectly fine IMO

Reply:
We lose the ability to rename things quickly via the language server by using a string literal. The difference between an enum and a string literal is DX and code-as-documentation. If we were to use the enum, we would make it a top-level export from the manager.

In practice, there's no difference and we don't need to make any changes here. Just figured I'd raise the question. 🙂

Reply:
I totally agree that if it's free, let's go for it. But when I exported it globally, I ended up with a failed build in Slice Machine UI. I didn't have time to check really why. Maybe you have an idea? I'm not sure we did that before.